Updated graphing distributions Docs page (#5067)
DataDog/documentation: content/en/developers/metrics/distributions.md

## Overview

> Distributions are a new metric type that aggregate the values that are sent from multiple hosts during a flush interval to measure statistical distributions across your entire infrastructure. Distribution metrics are designed to instrument logical objects, like services, independently from the underlying hosts, and solve the problem created by Agent-level aggregation.

Reviewer: ```suggestion
Distributions are a metric type that aggregate values sent from multiple hosts during a flush interval to measure statistical distributions across your entire infrastructure. Distribution metrics are designed to instrument logical objects, like services, independently from the underlying hosts, and solve the problem created by Agent-level aggregation.
```

### Customization of tagging

> This functionality allows you to control tagging for metrics for which host-level granularity is not necessary see the [Distribution Metric page][3] to learn more about our whitelist-based tagging control. Note: the exclusion of tags with `!` will not be accepted in this customization feature.

Reviewer: ```suggestion
This functionality allows you to control tagging for metrics where host-level granularity is not necessary. See the [Distribution Metric page][3] to learn more about whitelist-based tagging control. **Note**: The exclusion of tags with `!` is not accepted with this feature.
```

## Submission

> Note: As distributions are a new metric type, you should submit distributions under new metric names.

Reviewer: ```suggestion
Since distributions are a different metric type, any previously submitted metrics should be instrumented under new metric names.
```

Author: Not necessarily that any previously submitted metrics should be instrumented under new metric names...just that distributions shouldn't have the same name as previously submitted metric types. 

Might be easier to say Note: Different metric types in Datadog should be submitted under different metric names

Reviewer 2: updated
